### PR TITLE
[TEST-1198][TEST-1240] Backported Python 2.7 compatibility

### PR DESCRIPTION
**Pull request by carsongee**
Merging still won't work, but https://jenkins-test.int.datarobot.com/job/DataRobot_Test_Suite_testmon_Update/225/ is an attempt to at least make it through an update run

### CODE REVIEW

**Comment by carsongee** [comment]
Merging now works, and I backported the nice report header from the v0.9.x branch

**Comment by carsongee** [comment]
Oh, and the DR CI stuffs even though there aren't any tests to run, it still validates that it is an installable python package

File: DRCODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+/               @datarobot/test-auto
```
Comment (andrew-qa) [inline DRCODEOWNERS:1]: oops. we didn't have codeowners before.

File: README.rst [hunk 2]
```diff
@@ -16,4 +18,13 @@ Quickstart
     pytest --testmon
 
 
-To learn more about specifying multiple project directories and troubleshooting, please head to `testmon.org <https://testmon.org>`_
+To learn more about specifying multiple project directories, configuring “sticky” testmon and troubleshooting, please head to `testmon.org <https://testmon.org>`_
+
+DataRobot Release
+=================
+
+To upload a new version:
```
Comment (andrew-qa) [inline README.rst:26]: That's really good to have here, because it's not quite obvious.

**Review by andrew-qa — APPROVED** [review]
Looks quite compact.

LGTM

**Comment by datarobot-jarvis** [comment]
:dollar: Total cost of executing tests in Jenkins: $**0.02** USD.

[What is this all about?](http://bit.ly/2BahK4V)

**Pull request merged**